### ISSUE TITLE
Export issue: exports only the last slide

### ISSUE THREAD

**Issue by clemgrim** [FormidableLabs/spectacle#596]
- [x] I've searched open [issues](https://www.github.com/FormidableLabs/spectacle/issues) to make sure I'm not opening a duplicate issue
- [x] This issue not specific to `spectacle-boilerplate` (those issues should be opened [here](https://github.com/FormidableLabs/spectacle-boilerplate/issues/new)).
- [x] I have read through the [docs](https://formidable.com/open-source/spectacle/docs/) before asking a question
- [x] I am using the latest version of Spectacle

### Describe Your Environment

What version of Spectacle are you using? (can be found by running `npm list spectacle`)
5.1.0 (tested with 4.4.1 before, same result)

What version of React are you using? (can be found by running `npm list react`)
16.5.2

What browser are you using?
Chrome

What machine are you on?
Mac OS, with node 10 and yarn 1.9

### Describe the Problem
I have created a Spectacle presentation with 16 slides. Now I want to export it in PDF

**Expected behavior:**
It should create a PDF with 16 pages 

**Actual behavior:**
It creates a PDF with only one page (the last slide).
I tested with `http://localhost:49530/#/?export` and `spectacle-renderer -u http://localhost:49530`, same result
Note that I don't use port 3000 because it's already used.

### Additional Information
In my presentation, one slide is one component

```jsx
export default class Presentation extends React.Component {
  render() {
    return (
      <Deck
        transition={['zoom', 'slide']}
        transitionDuration={500}
        theme={theme}
      >
        <Home />
        <History />
        <Implementation />
      </Deck>
    );
  }
}
```

It works fine in the browser, I don't know if the issue is related to this organization...

**Comment by ebrillhart** [comment]
Hi @clemgrim thanks for letting me know that you're running into this! We'll investigate ASAP.

**Comment by ryan-roemer** [comment]
Suggest verifying against local host for our own demo _first_ w 1. Export to pdf in browser, 2. Using spectacle-renderer

If that doesn’t repro the issue, let’s then ask for a full non working sample to verify against

**Comment by ebrillhart** [comment]
I think this is connected to #586

**Comment by glennsayers** [comment]
I can reproduce this pretty reliably. It occurs for me when I import a slide from another file rather than having everything inline.

This works:
```
 <Deck theme={theme}>
   <Slide transition={["zoom"]} bgColor="primary">
     <Heading size={1} fit caps lineHeight={1}>
        React
     </Heading>
     <Image src={Logo} height={200} width={200}/>
   </Slide>
 </Deck>
```

This doesn't:

```
import Intro from './intro';

...

 <Deck theme={theme}>
   <Intro/>
 </Deck>
```

**Comment by sreimer15** [comment]
I would love to help out with this bug, if at all possible. Is there any information on what might be going wrong with this one? Thank you!

**Comment by kale-stew** [comment]
@sreimer15 it looks like it's related to a few other open issues we have surrounding Slide theming and modularization (see #678, #586). We would welcome your help if you can figure out what's going on!

**Comment by kale-stew** [comment]
Closing as outdated, but I do acknowledge this is a true issue in classic spectacle. Behavior when importing slides is just one of the critical issues we're facing with the current API. In our [rewrite](https://github.com/FormidableLabs/spectacle/issues/716), we are planning to handle the export differently, as well as subsets of Slides. I am making an internal note to circle back to this strange behavior once we have released the new codebase, and to make sure that we're testing for this usecase with our new export behavior. 🙂
